.github/workflows/all-pr-tests.yml

```diff
@@ -0,0 +1,42 @@
+name: "All Pull Request Tests"
+
+on:
+  pull_request:
+    branches:
+      - main
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  # We use a single job to ensure that all steps run in the same environment and
+  # reduce the number of minutes used.
+  pr-tests:
+    # Don't run on draft PRs
+    if: github.event.pull_request.draft == false
+    # Timeout after 10 minutes
+    timeout-minutes: 10
+    # Define a matrix of PHP/WordPress versions to test against
+    strategy:
+      matrix:
+        php: [8.1, 8.2, 8.3]
```

Comment:
No support for PHP 8.0 anymore or just not testing it anymore?

Reply:
Testkit doesn't support it and it isn't a supported PHP version anymore. I don't see a need to keep supporting it IMO.

Reply:
@srtfisher Could you update this too then: https://github.com/alleyinteractive/wp-bulk-task?tab=readme-ov-file#install